Sources/swift-container-plugin/Documentation.docc/build.md

```diff
@@ -0,0 +1,42 @@
+# Build and package your service
+
+## Overview
+
+`build-container-image` is a command plugin which takes care of building your service, packaging it in a container image and uploading it to a container registry, all in one command:
+
+```shell
+% swift package --swift-sdk x86_64-swift-linux-musl \
+        build-container-image --from swift:slim --repository registry.example.com/myservice
+```
+
+* The `--swift-sdk` argument specifies the Swift SDK with which to build the executable.   In this case we are using the Static Linux SDK, which was installed earlier, to build an statically-linked x86_64 Linux binary
+* The `--from` argument specifies the base image on which our service will run.   `swift:slim` is the default, but you can choose your own base image or use `scratch` if your service does not require a base image at all.
+* The `--repository` argument specifies where the plugin will upload our finished image.
+
+The plugin needs permission to connect to the network to publish the image to the registry:
+
+```
+Plugin ‘ContainerImageBuilder’ wants permission to allow all network connections on all ports.
+Stated reason: “This command publishes images to container registries over the network”.
+Allow this plugin to allow all network connections on all ports? (yes/no)
+```
```

Comment:
```suggestion
> ```
```

Reply:
^ I *think* this set of updates will put the whole text block of the error message into the "Note" field, but I'm not 100% sure...

Reply:
Yes, this worked: fixed in 2127832af5b080b2206b89a05c37921dafb4f56a